Document CSSFontFeatureValuesRule (#25570)
mdn/content: files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md

---
title: CSSFontFeatureValuesRule.fontFamily
slug: Web/API/CSSFontFeatureValuesRule/fontFamily
page-type: web-api-instance-property
browser-compat: api.CSSFontFeatureValuesRule.fontFamily
---

{{ APIRef("CSSOM") }}

> The **`fontFamily`** property of the {{domxref("CSSConditionRule")}} interface represent the name of the font family it applies to.

Reviewer: ```suggestion
The **`fontFamily`** property of the {{domxref("CSSConditionRule")}} interface represents the name of the font family it applies to.
```

## Value

A string.

## Examples

### Read font family

> In this example, we declared to {{cssxref("@font-feature-values")}} one fo the _Font One_ font family, and the other for _Font Two_. We then uses the CSSOm to read these font families, displaying them into the log.

Reviewer: ```suggestion
In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

> In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.

Reviewer 2: ```suggestion
In this example, we declare two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```
